apps/hash-api/src/graphql/resolvers/ontology/property-type.ts

```diff
@@ -134,7 +133,7 @@ export const getPropertyTypeResolver: ResolverFn<
     },
   );
 
-  return mapSubgraph(propertyTypeSubgraph);
+  return propertyTypeSubgraph as Subgraph;
```

Comment:
```suggestion
  return propertyTypeSubgraph as Subgraph<PropertyTypeRootType>;
```

Reply:
Applied in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)